exec/csnex/Exec.cs

```diff
@@ -0,0 +1,686 @@
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void CALLE()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void CALLX()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void CALLV()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+#endregion
+#region JUMP Opcodes
+        void JUMP()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void JF()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void JT()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void JFCHAIN()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void JUMPTBL()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void JNASSERT()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void RET()
+        {
+            ip++;
+        }
+#endregion
+#region Stack Handler Opcodes
+        void DUP()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void DUPX1()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void DROP()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void DROPN()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void SWAP()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+#endregion
+#region Construct Opcodes
+        void CONSA()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void CONSD()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+#endregion
+#region Exception Opcodes
+        void EXCEPT()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+#endregion
+#region Memory Opcodes
+        void ALLOC()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+
+        void RESETC()
+        {
+            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+        }
+#endregion
+#endregion
+
+        private int Loop()
+        {
+            while (ip < bytecode.codelen && exit_code == 0) {
+                switch (bytecode.code[ip]) {
```

Comment:
Can you cast the byte here to `Opcode` so you don't have to have a `(byte)` cast in front of each case?

Reply:
Fixed.